app/controllers/v1/users_controller.rb

```diff
@@ -38,6 +38,34 @@ def auth
       end
     end
 
+    def phone_auth
+      user = User.find_by(phone: params[:phone])
```

Comment:
@cjdenio if params[:phone] is nil here is this going to find a User in the db where the `phone` field is null?

Reply:
Great catch, should be fixed now